app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,70 @@
+<resources>
+    <string name="app_name">Dates d\'expiration des aliments</string>
+    <string name="app_name_short">Exp. des aliments</string>
+    <string name="n_days_ago">Il y a %1$d jours</string>
+    <string name="yesterday">Hier</string>
+    <string name="today">Aujourd/'hui</string>
```

Comment:
There is a typo here: `/` instead of `\`.

Don't create a new pull request, just add another commit to the branch linked to this pull request.

Reply:
Thanks for your suggestions I will send a new PR in some time. Sorry for the blunders.😅

Reply:
Are you sure to make a new pull request? You can save time by only add a commit here to fix the typo.

Reply:
Yeah sorry I meant that only. I will fix the code then commit it. 😅